tests/core/test_annotated_utterance.py

```diff
@@ -13,30 +15,68 @@ def test_utterance_text():
 def test_comparison():
```

Comment:
Can you add another case here, where there is a difference in the annotations (but not in the text or dialogue acts)?

Reply:
Done.